python/test/vmafossexec_test.py

```diff
@@ -929,15 +929,15 @@ def test_run_vmafossexec_runner_akiyo_multiply_no_enhn_gain_model_float(self):
 
         results = self.runner.results
 
-        try: self.assertAlmostEqual(results[0]['VMAFOSSEXEC_adm2_score'], 0.9574308606115118, places=4)  # 1.116691484215469
+        try: self.assertAlmostEqual(results[0]['VMAFOSSEXEC_adm2_egl_1.00_score'], 0.9574308606115118, places=4)  # 1.116691484215469
         except AssertionError as e: self.verificationErrors.append(str(e))
-        try: self.assertAlmostEqual(results[0]['VMAFOSSEXEC_vif_scale0_score'], 0.983699512450884, places=4)  # 1.0522544319369052
+        try: self.assertAlmostEqual(results[0]['VMAFOSSEXEC_vif_scale0_egl_1.00_score'], 0.983699512450884, places=4)  # 1.0522544319369052
         except AssertionError as e: self.verificationErrors.append(str(e))
-        try: self.assertAlmostEqual(results[0]['VMAFOSSEXEC_vif_scale1_score'], 0.9974276726830457, places=4)  # 1.0705609423182443
+        try: self.assertAlmostEqual(results[0]['VMAFOSSEXEC_vif_scale1_egl_1.00_score'], 0.9974276726830457, places=4)  # 1.0705609423182443
```

Comment:
Is the two decimal digit the normalized format for all parameters? Would it be better to normalize to `egl_1`?